msteams-platform/samples/code-samples.md

```diff
@@ -8,8 +8,11 @@ keywords: Microsoft teams developer samples
 
 These code samples show you the capabilities of Microsoft Teams apps and various ways to implement those capabilities:
 
+<<<<<<< HEAD
 * **[Get Started Sample](https://github.com/OfficeDev/microsoft-teams-sample-get-started)**&emsp;This sample shows all the capabilities available in a Microsoft Teams app, including bots, tabs, messaging extensions, and connectors. Source code is provided in both C# and Node.js.
 * **[Microsoft Graph API Samples](https://github.com/OfficeDev/microsoft-teams-sample-graph)**&emsp;These samples demonstrate using Microsoft Graph API calls to perform tasks such as querying teams and channels from a web service running outside Microsoft Teams.
+=======
```

Comment:
Same with this and the line below. Conflict markers. You should remove them while resolving the conflicts or during a merge.